src/Symfony/Component/Mailer/Bridge/Amazon/Transport/SesSmtpTransport.php

```diff
@@ -28,9 +28,13 @@ class SesSmtpTransport extends EsmtpTransport
     /**
      * @param string|null $region Amazon SES region
      */
-    public function __construct(string $username, string $password, string $region = null, EventDispatcherInterface $dispatcher = null, LoggerInterface $logger = null)
+    public function __construct(string $username, string $password, string $region = null, EventDispatcherInterface $dispatcher = null, LoggerInterface $logger = null, string $host = 'default')
     {
-        parent::__construct(sprintf('email-smtp.%s.amazonaws.com', $region ?: 'eu-west-1'), 465, true, $dispatcher, $logger);
+        if ('default' === $host) {
+            $host = sprintf('email-smtp.%s.amazonaws.com', $region ?: 'eu-west-1');
+        }
+
+        parent::__construct($host, 465, true, $dispatcher, $logger);
```

Comment:
the `port` could also be extracted from the DSN

Reply:
If I'm not mistaken, the port will always be `465` when using `ses-smtp`. At least I've not seen a way to change the port in a VPC endpoint for aws-ses.

Reply:
wait... Why are you trying to use a custom host?

If you target only AWS, the region is (should be) enough.

Reply:
oh, ok, it's for VPC endpoint.